app/bundles/PageBundle/Model/PageModel.php

```diff
@@ -1038,6 +996,77 @@ public function getPageList($limit = 10, \DateTime $dateFrom = null, \DateTime $
         return $q->execute()->fetchAll();
     }
 
+    /**
+     * Get all params (e.g. UTM tags) from a url.
+     */
+    protected function getQueryFromUrl(string $pageUrl): array
+    {
+        $query             = [];
+        $urlQuery          = parse_url($pageUrl, PHP_URL_QUERY);
+        parse_str($urlQuery, $urlQueryArray);
+
+        if (empty($urlQueryArray)) {
```

Comment:
Is this if needed? If `$urlQueryArray` is an empty array the the foreach loop won't do anything, right?

Reply:
Wanted to be extra careful. But yes, I can be removed.